src/array.ts

```diff
@@ -134,7 +134,7 @@ export namespace Shim {
 
 		while (count > 0) {
 			if (start in target) {
-				target[offset] = target[start];
+				(<any> target)[offset] = target[start];
```

Comment:
Unsure the correct approach for this. Without the cast to `<any>` this was causing a compilation error of `Left-hand side of assignment expression cannot be a constant or a read-only property.` 

Thoughts welcome 😄